adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBInterstitialRenderer.m

```diff
@@ -0,0 +1,93 @@
+//
+//  GADPangleRTBInterstitialRenderer.m
+//  PangleAdapter
+//
+//  Created by bytedance on 2022/1/11.
+//
+
+#import "GADPangleRTBInterstitialRenderer.h"
+#import "GADMediationAdapterPangleConstants.h"
+#import <BUAdSDK/BUAdSDK.h>
+#import "GADMAdapterPangleUtils.h"
+
+@interface GADPangleRTBInterstitialRenderer()<BUFullscreenVideoAdDelegate>
+
+@end
+
+@implementation GADPangleRTBInterstitialRenderer {
+    /// The completion handler to call when the ad loading succeeds or fails.
+    GADMediationInterstitialLoadCompletionHandler _loadCompletionHandler;
+    /// The Pangle Interstitial ad.
```

Comment:
Nit: `interstitial`.

Reply:
I don't understand, could you explain it?

Reply:
Nit, update to lower case letter: `/// The Pangle interstitial ad.`